mappings/net/minecraft/client/gui/GuiInputListener.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS csu net/minecraft/client/gui/GuiInputListener
```

Comment:
This doesn't purely handle input-related behaviour -- proposing something along the lines of `GuiComponent`.